1-Introduction/1-intro-to-ML/translations/README.es.md

```diff
@@ -0,0 +1,113 @@
+# Introducción al machine learning
+
+[![ML, IA, deep learning - ¿Cuál es la diferencia?](https://img.youtube.com/vi/lTd9RSxS9ZE/0.jpg)](https://youtu.be/lTd9RSxS9ZE "ML, IA, deep learning - ¿Cuál es la diferencia?")
+
+> 🎥 Haz clic en la imagen de arriba para ver un video donde se discuten las diferencias entre el machine learning, la inteligencia artificial, y el deep learning.
+
+## [Cuestionario previo a la conferencia](https://white-water-09ec41f0f.azurestaticapps.net/quiz/1/)
+
+### Introducción
+
+Bienvenid@ a este curso acerca del machine learning (ML) clásico para principiantes! Ya sea que tú seas completamente nuev@ en este tema, o un practicante experimentado del ML que busca refrescar los conocimientos en un área, ¡estamos contentos de que te nos unas! Queremos crear un punto de lanzamiento amigable para tus estudios de ML y nos encantaría evaluar, responder, e incorporar tu [retroalimentación](https://github.com/microsoft/ML-For-Beginners/discussions).}
+
+[![Introducción al ML](https://img.youtube.com/vi/h0e2HAPTGF4/0.jpg)](https://youtu.be/h0e2HAPTGF4 "Introducción al ML")
+
+> Haz clic en la imagen de arriba para ver el video: John Guttag del MIT presenta el machine learning
+
+### Empezando con el machine learning
+
+Antes de comenzar con este currículum, debes tener tu computadora configurada y lista para ejecutar los notebooks localmente.
+
+- **Configura tu equipo con estos videos**. Aprende más acerca de como configurar tu equipo con [estos videos](https://www.youtube.com/playlist?list=PLlrxD0HtieHhS8VzuMCfQD4uJ9yne1mE6).
+- **Aprende Python**. También se recomienda que tengas un entendimiento básico de [Python](https://docs.microsoft.com/learn/paths/python-language/?WT.mc_id=academic-15963-cxa), un lenguaje de programación útil para los científic@s de datos, y que se utiliza en este curso.
```

Comment:
One of the problems of attempting to be neutral (or inclusive) in Spanish is that you get these oddities on the articles preceding the noun. So instead of:

> los cientific@s de datos

It would be more correct to do:

> l@s cientific@s de datos

I don't think that using `@` is the right path forward

Reply:
Do you have any sug inclusive language? I know english isn't super gendered but in spanish a lot of words typically use the male gender, and that rubs me the wrong way a bit.

Reply:
I don't have any suggestions. I agree that Spanish usually uses the male gender, so I tend to go with that as well.